DigitalLearningSolutions.Web/Controllers/TrackingSystem/CourseSetup/AdminFieldsController.cs

```diff
@@ -140,11 +146,47 @@ BulkAdminFieldAnswersViewModel model
             );
         }
 
+        [HttpGet]

```

Comment:
Could we add a few tests for this? (e.g. redirect vs returning a view, not removing admin fields without confirmation)

Reply:
Good idea, created four tests - 
- RemoveAdminField_removes_admin_field_with_no_user_answers
- RemoveAdminField_returns_remove_view_if_admin_field_has_user_answers
- RemoveAdminField_does_not_remove_admin_field_without_confirmation
- RemoveAdminField_removes_admin_field_with_confirmation_and_redirects